src/ConfigReader/ConfigDefaults.cs

```diff
@@ -17,7 +17,8 @@ public static ConfigResult Convert(ConfigInput? fileConfig, ConfigInput otherCon
                 Header = otherConfig.Header,
                 TocExcludes = otherConfig.TocExcludes,
                 UrlsAsSnippets = otherConfig.UrlsAsSnippets,
-                TocLevel = otherConfig.TocLevel.GetValueOrDefault(2)
+                TocLevel = otherConfig.TocLevel.GetValueOrDefault(2),
+                TreatMissingSnippetsAsErrors = otherConfig.ReadOnly.GetValueOrDefault()
```

Comment:
shouldnt the default be "true"? as in not an change in behavior. aslo why is it `ReadOnly`?

Reply:
yeah. if I wanted to change the default value, I should've asked you / suggested instead of switching it here. 
After thinking it through the original default you have is good.